posthog/session_recordings/session_recording_api.py

```diff
@@ -276,7 +279,10 @@ def safely_get_object(self, queryset) -> SessionRecording:
 
     def list(self, request: request.Request, *args: Any, **kwargs: Any) -> Response:
         filter = SessionRecordingsFilter(request=request, team=self.team)
-        return list_recordings_response(filter, request, self.get_serializer_context())
+        try:
+            return list_recordings_response(filter, request, self.get_serializer_context())
+        except (ExposedHogQLError, ExposedCHQueryError) as e:
+            raise ValidationError(str(e), getattr(e, "code_name", None))
```

Comment:
I kept the section that raises ValidationError because I believe ExposedHogQLError and ExposedCHQueryError are client-side errors. Therefore, a 4xx HTTP status code, particularly 400 (bad request), seems more fitting than the 500 (internal server error) status code that was originally being used.

Please let me know if this assumption is incorrect and if I should adjust or remove this implementation